aspnetcore/tutorials/first-mvc-app/working-with-sql.md

```diff
@@ -1,28 +1,34 @@
 ---
 title: Part 5, work with a database in an ASP.NET Core MVC app
 author: rick-anderson
-description: Part 5, add a model to an ASP.NET Core MVC app
+description: Part 5 of tutorial series on ASP.NET Core MVC.
 ms.author: riande
-ms.date: 11/10/2020
+ms.date: 02/16/2020
 no-loc: [appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
 uid: tutorials/first-mvc-app/working-with-sql
 ---
+
 # Part 5, work with a database in an ASP.NET Core MVC app
 
 ::: moniker range=">= aspnetcore-3.0"
 
-By [Rick Anderson](https://twitter.com/RickAndMSFT)
+The `MvcMovieContext` object handles the tasks of:
 
-The `MvcMovieContext` object handles the task of connecting to the database and mapping `Movie` objects to database records. The database context is registered with the [Dependency Injection](xref:fundamentals/dependency-injection) container in the `ConfigureServices` method in the *Startup.cs* file:
+* Connecting to the database.
+* Mapping `Movie` objects to database records.
+
+The database context is registered with the [Dependency Injection](xref:fundamentals/dependency-injection) container in the `ConfigureServices` method of the *Startup.cs* file:
 
 # [Visual Studio](#tab/visual-studio)
 
 [!code-csharp[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie3/Startup.cs?name=snippet_ConfigureServices&highlight=5-6)]
 
-The ASP.NET Core [Configuration](xref:fundamentals/configuration/index) system reads the `ConnectionString`. For local development, it gets the connection string from the *appsettings.json* file:
+The ASP.NET Core [Configuration](xref:fundamentals/configuration/index) system reads the `ConnectionString` key. For local development, it gets the connection string from the *appsettings.json* file:
 
 [!code-json[](start-mvc/sample/MvcMovie/appsettings.json?highlight=2&range=8-10)]
 
+In the preceding connection string example, `Server=(localdb)` prompts the necessary SQL Server Express LocalDB infrastructure to be created and started automatically.  SQL Server Express LocalDB is installed with Visual Studio by default.
```

Comment:
removed this. The connection string doesn't do any of that. It's explained below.